src/d/actor/d_a_sail.cpp

```diff
@@ -449,39 +498,445 @@ void daSail_packet_c::setNrmMtx() {
+    mDoMtx_copy(*calc_mtx, i_this->mSailPacket.m1C3C->mBaseTransformMtx);
+
+    MtxTrans(0.0f, 0.0f, 0.0f, 0);
+    mDoMtx_copy(*calc_mtx, i_this->mSailPacket.m0040);
+
+    i_this->mSailPacket.mA0 = &i_this->tevStr;
+
+    j3dSys.getDrawBuffer(0)->entryImm(&i_this->mSailPacket, 0);
+
+    return TRUE;
 }
 
 /* 000013CC-00001CEC       .text sail_pos_move__FP10sail_class */
 static void sail_pos_move(sail_class* i_this) {
-    /* Nonmatching */
+    cXyz* windVec = dKyw_get_wind_vec();
+    s16 windAngle = cM_atan2s(windVec->x, windVec->z);
+    mDoMtx_YrotS(*calc_mtx, -(i_this->current.angle.y - windAngle));
+
+    cXyz sp28;
+    cXyz sp1C;
+    cXyz sp10;
+
+    sp28.set(0.0f, 0.0f, 0.064f);
+    MtxPosition(&sp28, &sp1C);
+
+    f32 f31 = (0.9f * std::fabsf(sp1C.z)) + 0.02f + (0.064f / 10.0f);
+    sp28.x = 0.0f;
+    sp28.z = 1.0f;
+    MtxPosition(&sp28, &sp1C);
+
+    f32 f25 = (0.9f * std::fabsf(sp1C.z)) + 0.1f;
+    f32 f0 = std::fabsf(sp1C.x);
+
+    i_this->mSailPacket.m1C40 += (s32)((1.0f - (i_this->mSailPacket.m1C44 * 0.95f)) * 4000.0f) * ((f25 * 0.85f) + 0.15f);
+    i_this->mSailPacket.m1C50 = ((f0 * 1.5f) + 1.0f) * 7500.0f;
+    i_this->mSailPacket.m1C52 = ((f25 * 0.95f) + 0.05f) * 7200.0f;
+
+    cXyz* vtxPos = i_this->mSailPacket.get_pos(0);
+    f32 sp6C[7] = {};
+    f32 sp34[7] = {};
+
+    f25 *= 1.0f - (0.5f * i_this->mSailPacket.m1C44 * i_this->mSailPacket.m1C4C);
+    f0 *= 1.0f - (0.75f * i_this->mSailPacket.m1C44);
+
+    f32 ELEVEN = 11.0f; // this must be non-const
+
+    for (int i = 0; i < 12; i++) {
+        f32 f12 = i - 6;
+        f32 f13 = i - 5.5f;
+
+        sp10.z = f25 * (-ELEVEN * f12 * f12 + 396.0f);
+
+        for (int j = 0; j < 7; j++, vtxPos++) {
+            f32 f24;
+            f32 f23 = j - 3;
+            f32 f22, f21, f20; // these are xyz components
+            f32 f17 = i * 5.0f;
+
+            u32 r22 = (i_this->mSailPacket.m1C50 * (s32)f23) + (i_this->mSailPacket.m1C52 * (s32)f12);
+
+            f20 = 1750.0f * ((f0 * f0) + 0.05f) * JMASSin(-i_this->mSailPacket.m1C40 + r22) * f31;
+            f22 = 550.0f * ((f0 * f0) + 0.1f) * JMASCos(i_this->mSailPacket.m1C40 + r22) * f31;
+            f21 = std::sqrtf((f22 * f22) + (f20 * f20)) * 0.1f;
+
+            f24 = (f25 * ((10.0f * f17) + (-f17 * f23 * f23)));
+            f24 += ((sp10.z * (18.0f - f23 * f23)) / 18.0f);
+
+            f32 f15 = f24 - sp34[j];
+
+            f32 f16;
+            if (f15 > 100.0f) {
+                f16 = 100.0f;
+                sp34[j] += 100.0f;
+            } else {
+                f16 = -100.0f;
+                if (f15 < f16) {
+                    sp34[j] -= 100.0f;
+                } else {
+                    f16 = f15;
+                    sp34[j] = f24;
+                }
+            }
+
+            f32 f14 = 100.0f - std::sqrtf((125.0f * 125.0f) - (f16 * f16));
+            sp6C[j] += f14 > -10.0f ? f14 : -10.0f; // this looks like cLib_minLimit but it does not match
```

Comment:
```suggestion
            sp6C[j] += f14 > -10.0f ? f14 : -10.0f; 
```
template functions like cLib_minLimit appear in the debug map for every unit they were used in (unlike regular functions that appear for just the first one). cLib_minLimit doesn't appear for d_a_sail.o, so they didn't use it here, and must have typed this manually